test/e2e/tools/tools_test.go

```diff
@@ -25,12 +27,16 @@ import (
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
 
+	"k8s.io/klog/v2"
+
 	"github.com/k8stopologyawareschedwg/deployer/pkg/deployer/platform"
+
+	"github.com/openshift-kni/numaresources-operator/internal/api/features"
 )
 
-var _ = Describe("[tools] Auxiliary tools", func() {
+var _ = Describe("[tools] Auxiliary tools", Label("tools"), func() {
 	Context("with the binary available", func() {
-		It("[lsplatform] lsplatform should detect the cluster", func() {
+		It("[lsplatform] lsplatform should detect the cluster", Label("inspectfeatures"), func() {
```

Comment:
this specific test is not pertaining inspect features though (it predates the change)

Reply:
by mistake, the label should be lsplatform